### PR TITLE
[DOCS-7799] Network path private beta

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
Adding private beta docs for Network Path

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/network_monitoring/network_path/list_view.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: List View
+kind: documentation
+description: Investigate the Network Path List View
+further_reading:
+- link: "/network_monitoring/network_path/path_view"
+  tag: "Doc"
```
Comment (alai97) [inline content/en/network_monitoring/network_path/list_view.md:7]:
```suggestion
  tag: "Documentation"
```

File: content/en/network_monitoring/network_path/_index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: Network Path
+kind: documentation
+description: Investigate network traffic paths 
+further_reading:
+- link: "/network_monitoring/network_path/list_view"
+  tag: "Doc"
```
Comment (alai97) [inline content/en/network_monitoring/network_path/_index.md:7]:
```suggestion
  tag: "Documentation"
```

File: content/en/network_monitoring/network_path/_index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  text: "Learn more about the List View in Network Path"
+- link: "/network_monitoring/network_path/path_view"
+  tag: "Doc"
```
Comment (alai97) [inline content/en/network_monitoring/network_path/_index.md:10]:
```suggestion
  tag: "Documentation"
```

**Review by alai97 — COMMENTED** [review]
Looks good, some copy suggestions.

File: content/en/network_monitoring/network_path/_index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Overview
+
+[Network Path][1] illustrates the route that network traffic follows from its origin to its destination. This provides network administrators with the capability to precisely identify the source of network problems, whether they're internal or from an Internet Service Provider (ISP), or due to other issues such as mis-routing. Each row signifies a route from a source to it's destination, as depicted in the `source` and `destination` facet panel.
+
+{{< img src="network_performance_monitoring/network_path/network_path_default_view.png" alt="The Network Path default view, sorting by Datacenter and Un-Grouped Destination" >}}
+
+## How it works
+
+Datadog performs a `traceroute` at the host level to illustrate the packet's journey and latency at each hop from a source to it's destination. Each host runs its own `traceroute`, and the path shown is a visual representation of this list. The following diagram depicts the typical flow of a network path from a source (Host) to it's destination.
```
[the remark this answers is not in the document]
Comment (aliciascott) [reply content/en/network_monitoring/network_path/_index.md]: Fixed this as well

**Comment by aliciascott** [comment]
This PR contained a conflict with the build_preview I wasn't able to fix, hence closing this and replaced with a new PR:

https://github.com/DataDog/documentation/pull/23772

**Pull request closed without merging**